From 6e1b280185c5c3d4f408305a4b65b669857e6304 Mon Sep 17 00:00:00 2001
From: Shawn Ferry <sferry@gmail.com>
Date: Wed, 16 Aug 2017 17:01:32 -0400
Subject: [PATCH] (MODULES-5497) puppetlabs-concat : the OS formerly known as
 Solaris 12

Solaris 12 is now Solaris 11.4 metadata.json needs an update

patch submitted upstream
https://tickets.puppetlabs.com/browse/MODULES-5497
---
 metadata.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- puppetlabs-concat-9.0.1/metadata.json.orig	2024-01-15 22:00:48.920503327 -0800
+++ puppetlabs-concat-9.0.1/metadata.json	2024-01-15 22:01:18.731904702 -0800
@@ -66,7 +66,8 @@
     {
       "operatingsystem": "Solaris",
       "operatingsystemrelease": [
-        "11"
+        "11",
+        "11.4"
       ]
     },
     {
